src/config.js

```diff
@@ -158,12 +158,11 @@ module.exports = (common) => {
       describe('.set', () => {
         it('set a new key', () => {
           return ipfs.config.set('Fruit', 'banana')
-            .then(() => {
-              ipfs.config.get('Fruit', (err, fruit) => {
-                expect(err).to.not.exist
-                expect(fruit).to.equal('banana')
-              })
+            .then(() => ipfs.config.get('Fruit'))
+            .then((fruit) => {
+              expect(fruit).to.equal('banana')
             })
+            .catch((err) => expect(err).to.not.exist)
```

Comment:
Shouldn't need the catch here since we're returning the promise so mocha should automatically catch any errors.